rollup.config.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
are you using the generator? these options had changed

https://github.com/cheminfo/generator-cheminfo/issues/11

Reply:
@maasencioh you need to release a new version of the generator

Reply:
@targos done, thanks